game_eggs/terraria/vanilla/egg-terraria-vanilla.json

```diff
@@ -80,7 +85,38 @@
             "default_value": "Welcome!",
             "user_viewable": true,
             "user_editable": true,
-            "rules": "required|string|max:128"
+            "rules": "required|string|max:128",
+            "field_type": "text"
+        },
+        {
+            "name": "World Seed",
+            "description": "The Seed to use when creating the World",
+            "env_variable": "WORLD_SEED",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Password",
+            "description": "The password which should be used.",
+            "env_variable": "PASSWORD",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string",
+            "field_type": "text"
+        },
+        {
+            "name": "NPCStream",
+            "description": "Reduces enemy skipping but increases bandwidth usage. The lower the number the less skipping will happen, but more data is sent. 0 is off.",
+            "env_variable": "NPCSTREAM",
```

Comment:
What is the default value for this? So far as I can tell you are just forcing a value that drives higher bandwidth.

Reply:
The default value should be 60 (thats the value which is set in an default serverconfig.txt, but is commented out) but will result in bad enemy lagging, from my experience.
A value of 0 resulted, in my testing, with 1 Player connected to the server, to a maximum bandwidth usage of about 8 kB/s outbound traffic, inbound traffic did not change noticable.
Thats the reason why i defaulted to 0

Reply:
That's reasonable